openapi-bal-service/build.gradle

```diff
@@ -31,6 +31,7 @@ configurations {
 }
 
 dependencies {
+    implementation project(':openapi-validator')
```

Comment:
Let's use the semantic API to resolve the absolute paths, instead of using the validator as a dependency for the same purpose.